index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+  </head>
+  <body>
+    <div id="main">
```

Comment:
Let's make the page more semantic, using a [`main` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) here would be better.